src/toil_scripts/adam_uberscript/adam_uberscript.py

```diff
@@ -119,8 +118,13 @@ def launch_pipeline(params):
                                '-o', 'UserKnownHostsFile=/dev/null',
                                '-o', 'StrictHostKeyChecking=no',
                                'screen', '-dmS', params.cluster_name])
-        # Run command on screen session
-        
+        # Run command on screen session  
+
+        if params.reference_genome == 'GRCh38':
+            from toil_scripts.adam_uberscript.input_files import GRCh38_inputs as inputs
+        if params.reference_genome == 'hg19':
+            from toil_scripts.adam_uberscript.input_files import hg19_inputs as inputs       
```

Comment:
This gets validated during argument parsing. See https://github.com/BD2KGenomics/toil-scripts/pull/133/files#diff-7060afbea50ff85e89fe57836bc27314R451.